docs/en/quantization/w4a16.md

```diff
@@ -128,3 +128,8 @@ We benchmarked the Llama-2-7B-chat and Llama-2-13B-chat models with 4-bit quanti
 | ---------------- | ------- | ------- | --------- |
 | Llama-2-7B-chat  | 112.9   | 159.4   | 206.4     |
 | Llama-2-13B-chat | N/A     | 90.7    | 115.8     |
+
+## FAQs
+
+1. Out of Memory error during quantization due to insufficient GPU memory: This can be addressed by reducing the parameter `--calib-seqlen`, increasing the parameter `--calib-samples`, and set `--batch-size` to 1.
+2. NaN value error during quantization: This may be due to a large number of zeros in the weights. For a solution, refer to the modifications in https://github.com/InternLM/lmdeploy/pull/1844/files.
```

Comment:
I think we should address the NaN issue. Giving users an option to clamp the value is a solution as long as the root cause is massive zeros in the weights